source/community/get-involved/resources/slide-decks.html.md

```diff
@@ -95,11 +95,11 @@ Arch & Work in progress: [(PDF)](/w/images/b/be/Ovirt-arch-iheim.pdf)
 
 oVirt Node: [(PDF)](/wp-content/uploads/2011/11/ovirt-node.pdf)
 
-oVirt API: ![](OVirt-API-CLI-SDK-20111102.pdf  "fig:OVirt-API-CLI-SDK-20111102.pdf ") (PDF)
+oVirt API: [(PDF)](/images/wiki/OVirt-API-CLI-SDK-20111102.pdf)
```

Comment:
Much better, but odd that there's [()] — perhaps we should do another pass and remove the () from links? (Stylistic, not functional suggestion.)

Same goes for any below that also have the odd [()] construct.